packages/components/src/spec-distributions.json

```diff
@@ -32,89 +32,152 @@
       }
     }
   ],
-
   "scales": [
     {
       "name": "xscale",
       "type": "pow",
-      "exponent": { "signal": "xscale ? 0.1 : 1" },
+      "exponent": {
+        "signal": "xscale ? 0.1 : 1"
```

Comment:
Is this you doing the formatting here or prettier? I'm pretty sure I run prettier before I commit... So this might just change back the next time I edit. Either way, have you run prettier?

Reply:
I just checked; I was using a different formatter before ("Json language features"; I assume a default in VSCode). I changed to Prettier and run it; it changed a very few things, but it didn't change it back to how it was before.

Reply:
Yay! Thanks for fixing that up